.github/workflows/deploy-web-components-to-canary.yml

```diff
@@ -20,11 +20,18 @@ jobs:
         uses: ./.github/actions/install-ibmcloud-cli
       - name: Build project
         run: yarn build
-      - name: Building @carbon/ibmdotcom-web-components storybook
+      - name: Create .env file
         env:
-          CORS_PROXY: ${{ secrets.CORS_PROXY }}
           KALTURA_PARTNER_ID: ${{ secrets.KALTURA_PARTNER_ID }}
           KALTURA_UICONF_ID: ${{ secrets.KALTURA_UICONF_ID }}
+        run:
+          touch .env
```

Comment:
If this works, we would have to make adjustments to the other yml file and jobs I created right?

Reply:
Yes, I think so.